fastlane/lanes/release.rb

```diff
@@ -60,10 +62,12 @@
     update_frozen_strings_for_translation
 
     ensure_git_status_clean
-    push_to_git_remote
+    push_to_git_remote(tags: false)
 
     new_version = android_get_app_version()
     trigger_beta_build(branch_to_build: "release/#{new_version}")
+
+    create_release_management_pull_request('trunk', "Merge #{new_version} code freeze to trunk")
```

Comment:
Nitpick. Since we dropped the `develop`-`trunk` setup, I stopped adding "to trunk" in my PRs because that's the branch where usually release PRs land.

```suggestion
    create_release_management_pull_request('trunk', "Merge #{new_version} code freeze")
```

Reply:
With release management in CI, not all PRs target the `trunk` branch. Some of them target the release branch, so I think it's useful to have this information.